src/code.md

```diff
@@ -204,4 +204,49 @@ match s.strip_prefix("0x") {
 
 ## How do I access variables from within a spawned thread?
 
-Use [`crossbeam_utils::thread::scope`](https://docs.rs/crossbeam/0.8.1/crossbeam/thread/struct.Scope.html#method.spawn).
\ No newline at end of file
+Use [`std::thread::scope`](https://doc.rust-lang.org/nightly/std/thread/fn.scope.html).
+
+## When should I use runtime checks vs jumping through hoops to do static checks?
+
+Everyone learns Rust a different way, but it's said that some people reach a
+point of "trait mania" where they try to encode _too much_ via the type
+system, and get in a mess. So, in learning Rust, you will want to strike a
+balance between runtime checks (easy) or static compile-time checks (more
+efficient but requires deeper understanding.)
+
+> It’s very personal - some people learn better if they opt out of
+> language features, others not - MG.
```

Comment:
But should we share a goal of eventually getting to maximizing static checking, or no? (I assume yes, but that may be my bias.)

Reply:
Hm. I think I aspire to a higher level of static checking than even most Rustaceans.

For example my fundamental `Api` structure [in autocxx](https://github.com/google/autocxx/blob/main/engine/src/conversion/api.rs#L462) is parameterized to the point that it's definitely a bit annoying. I do it this way because I don't trust myself and I want to minimize opportunity for runtime crashes when I am later incompetent.

But, I can tell that I'm straining at the boundaries of the language - in this case, most of the enum variants are not parameterized and won't change at all based on the `T` type parameter - yet there's no way to handle them nicely and lots of copying is required as the `T` type changes.

So, I do think there's probably a level at which the language starts to hate you for trying to do _too much_ statically.

In any case I'm happy with MG's quote here. I think it's valuable to be inclusive of different learning styles. So I think I'll keep this as-is.